internal/errgroup/group_test.go

```diff
@@ -511,102 +586,58 @@ func Test_group_Go(t *testing.T) {
 			if test.checkFunc == nil {
 				test.checkFunc = defaultCheckFunc
 			}
-			g := &group{
```

Comment:
I think it is better than `generator`, even it has lint error. what do you think?